picard/album.py

```diff
@@ -346,7 +352,11 @@ def _release_request_finished(self, document, http, error):
         parse_result = None
         try:
             if error:
-                self.error_append(http.errorString())
+                err_str = http.errorString() if hasattr(http, 'errorString') else str(error)
+                user_msg = ("Das Laden der Album-Informationen ist fehlgeschlagen. "
+                            "Bitte prüfen Sie Ihre Internetverbindung oder versuchen Sie es später erneut.\n\n"
+                            f"[Technischer Fehler: {err_str}]")
+                self.error_append(user_msg)
```

Comment:
In principle I am supportive of an improvement to this error message - it would be beneficial to add a description saying that it is an http error, as well as the contents of the actual error message, but as previously stated error messages need to be in English, though they can be translated - and in any case this isn't the correct way to insert parameters into strings.